### PR TITLE
simplistic implementation for #111 (show documentation)

### PR DESCRIPTION
**Pull request by stef-k**
A very simple idea to use for documentaion tips. Uses the `go doc` command line tool.

Currently accessible from the command palette (ctrl + shift + p)

std lib package documentation:

std_lib_package_documentation

std lib method documentation:

std_lib_method_doc

project documentation:

project_doc

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
+100

File: gotools_documentation.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                p = os.path.basename(
+                    package.strip('\"'))
+
+                out, err = self.godoc(p + '.' +txt)
+                if out != '':
+                    break
+
+        html += "<p>"
+        html += out
+        html += "</p></div>"
```
Comment (stevekuznetsov) [inline gotools_documentation.py]: We may want to import `html`:
```python
>>> from html import HTML
>>> h = HTML()
>>> h.p('Hello, world!')
>>> print h
<p>Hello, world!</p>
```

**Comment by ironcladlou** [comment]
This is a great idea. Thanks for taking the time to work on the it.

I've been following the Sublime minihtml/popup stuff for a while, and am pretty bummed that it doesn't seem to be getting more attention/expansion. Once I get a little more comfortable with `go doc` nuances, the Sublime popup renderer/behavior, and your implementation, I'll have some more feedback. Probably in the form of PRs to your fork.

Thanks again, this will be a nice jumpstart for the feature.

**Comment by ironcladlou** [comment]
I've decided to go ahead and discontinue development of GoTools, so am closing the outstanding PRs. Sorry about that, and thanks for your contributions.

**Pull request closed without merging**